cwl_utils/docker_extract.py

```diff
@@ -32,6 +32,11 @@ def arg_parser() -> argparse.ArgumentParser:
         help="Use singularity to pull the image",
         action="store_true",
     )
+    parser.add_argument(
+        "--user-space-docker-cmd",
+        dest="user_space_docker_cmd",
```

Comment:
```suggestion
        "--container-engine",
```
No reason to mention "user space"

Reply:
Gotcha. I was just trying to keep it consistent with [cwltool](https://github.com/common-workflow-language/cwltool/blob/main/cwltool/argparser.py#L433) and [cwltoil](https://github.com/DataBiosphere/toil/blob/master/src/toil/cwl/cwltoil.py#L3179), but I can make the change.

Reply:
Done!